selfdrive/test/process_replay/process_replay.py

```diff
@@ -338,9 +339,9 @@ def replay_process_with_name(name, lr, *args, **kwargs):
   return replay_process(cfg, lr, *args, **kwargs)
 
 
-def replay_process(cfg, lr, fingerprint=None, return_all_logs=False):
+def replay_process(cfg, lr, fingerprint=None, return_all_logs=False, disable_pbar=False):
```

Comment:
didn't know what a pbar was
```suggestion
def replay_process(cfg, lr, fingerprint=None, return_all_logs=False, display_progress=False):
```